dashboard/Dockerfile

```diff
@@ -3,9 +3,11 @@ FROM python:latest
 WORKDIR /app
 
 COPY requirements.txt .
-RUN pip install -r requirements.txt
+RUN pip3 install -r requirements.txt
```

Comment:
I like that you added 'pip3' just to be sure

Reply:
Yeah, weird "fix". Either way it was safer.